src/Http/RequestFactory.php

```diff
@@ -81,7 +81,7 @@ public function createHttpRequest()
 		}
 
 		// path & query
-		$requestUrl = isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : '/';
+		$requestUrl = isset($_SERVER['REQUEST_URI']) ? Strings::replace($_SERVER['REQUEST_URI'], '#^\w+://[^/]+#i') : '/';
```

Comment:
I think it's better to make it more readable by splitting it into lines.

~~~php
$requestUrl = isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : '/';
$requestUrl = Strings::replace($requestUrl, '#^\w+://[^/]+#i');
~~~

The pattern could be optimized by removing pointless case-insensitive flag and avoiding any possible backtracking

~~~php
$requestUrl = isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : '/';
$requestUrl = Strings::replace($requestUrl, '#^\w++://[^/]++#');
~~~

With removed backtracking, we may probably replace `Strings::replace` with faster `preg_replace`

~~~php
$requestUrl = isset($_SERVER['REQUEST_URI']) ? $_SERVER['REQUEST_URI'] : '/';
$requestUrl = preg_replace('#^\w++://[^/]++#', '', $requestUrl);
~~~